src/commands/cancel.ts

```diff
@@ -31,24 +32,28 @@ export default class Cancel extends CommandWithCatch {
       description: 'ID of reservation to cancel, or a list of IDs.',
       required: true,
     }),
+    ...pick(baseOptions, 'confirm'),
   };
 
   async run() {
     const { flags } = this.parse(Cancel);
     const ids = parseValueOrValues(flags.id);
 
-    const resToCancel = (await GET.schedule({ ids })).filter(
-      r => r.status === 'ACTIVE',
-    );
-    if (resToCancel.length === 0) {
-      this.logErrorAndExit('reservation(s) found, but none that are active');
-    }
+    if (!flags.confirm) {
```

Comment:
I think this error checking shouldn't be tied to the `--confirm` flag.

Reply:
👌